src/scenarios/UK.ts

```diff
@@ -0,0 +1,47 @@
+    importedCasesPerDay: 0.1,
+    hospitalCapacity: 128935, // 250k total hospital beds source: https://www.nuffieldtrust.org.uk/resource/hospital-bed-occupancy#:~:text=Between%20Q1%202010%2F11%20and,%25%20in%20Q4%202019%2F20.
+    runUpPeriod: [], // Disable run-up period
+    gdpPerDay: GDP_Uk / 365.0,
+    power: 1,
+    distr_family: 'nbinom',
+    dynamics: 'SIS',
+    mortality: 0.01,
+    time_lumping: false,
+    initialContainmentPolicies: [],
+    initialCapabilityImprovements: [],
+    baseImmunity: 0.1086,
+    initialPublicSupport: 50,
+    dailyIncreaseInImmunity: 0.0022
```

Comment:
Immunity (for vaccination model) and starting public support defined at scenario level